shared/output/cloud/language_v1/lib/google/cloud/language/v1/language_service/client.rb

```diff
@@ -152,8 +164,9 @@ def initialize
               credentials = @config.credentials
               # Use self-signed JWT if the endpoint is unchanged from default,
               # but only if the default endpoint does not have a region prefix.
```

Comment:
Not in this PR, but this assumption that region prefixes impact self-signed jwt in any way might be obsolete. Where is this coming from?

Reply:
This was a constraint at one point. I'm not sure if it still holds—you're right that it might be obsolete. I'll open a separate issue to investigate.

Reply:
:+1: